Real-Dev-Squad/website-dashboard
# Bug fix/search in groups

## What type of PR is this? (check all applicable)


## Description
When sharing a URL of groups like https://dashboard.realdevsquad.com/groups/?quiz-app, the search functionality is not working as expected. It highlights the matching group, but doesn't populate the search value in the input field or filter the group list. Additionally, typing in the search input field doesn't update the URL with the search value.

## QA Instructions, Screenshots, Recordings

Visit https://dev.realdevsquad.com/groups/index.html?dev=true&DSA. Confirm that the search value populates and the group list filters. Note that the URL updates are based on the input field.

## Added/updated tests?

      have not been included_
